.github/workflows/unit-tests.yml

```diff
@@ -43,18 +59,17 @@ jobs:
 
   coverage:
     name: Code coverage
-    needs:
-      - test
     runs-on: ubuntu-latest
+    needs: check_docs_only
```

Comment:
@camilamacedo86 Now the PR is working, I was missing this statement. Just wanted to ask should we remove statements 46-47 where we declared needs to `test`, since it was not used anywhere so I removed it, is it reasonable?